examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -0,0 +1,1191 @@
+ax1.legend()
+ax2.hist(
+    hist_data["p05"], bins=30, ec="black", color="cyan", alpha=0.4, label="Uninformative Prior"
+)
+ax2.hist(
+    hist_data_info["p05"], bins=30, ec="black", color="pink", alpha=0.4, label="Informative Prior"
+)
+ax2.legend()
+ax2.set_title("Distribution of 5% failure Time", fontsize=20)
+wbf = WeibullFitter().fit(item_period_bearing_cage["t"] + 1e-25, item_period_bearing_cage["failed"])
+wbf.plot_cumulative_density(ax=ax, color="cyan", label="Weibull MLE Fit")
+ax.legend()
+ax.set_ylim(0, 0.25);
+```
+
+We can see here how the Bayesian uncertainty estimates driven by our deliberately vague priors encompasses more uncertainty than our MLE fit and the uninformative prior implies a wider predictive distribution for the 5% and 10% failure times. The Bayesian model with uninformative priors seems to do a better job of capturing the uncertainty in the non-parametric estimates of our CDF, but without more information it's hard to tell which is the more appropriate model.
+
+The concrete estimates of failure percentage over time of each model fit are especially crucial in a situation where we have sparse data. It is a meaningful sense check that we can consult with subject matter experts about how plausible the expectation and range for the 10% failure time is for their product.
+
++++
+
+## Predicting the Number of Failures in an Interval
+
+Because our data on observed failures is extremely sparse, we have to be very careful about extrapolating beyond the observed range of time, but we can ask about the predictable number of failures in the lower tail of our cdf. This provides another view on this data which can be helpful for discussing with subject matters experts. 
+
+### The Plugin Estimate
+
+Imagine we want to know how many bearings will fail between 150 and 600 hours based of service. We can calculate this based on the estimated CDF and number of new future bearings. We first calculate: 
+
+$$ \hat{\rho} = \dfrac{\hat{F}(t_1) - \hat{F}(t_0)}{1 - \hat{F}(t_0)} $$
+
+to establish a probability for the failure occurring in the interval and then a point prediction for the number of failures in the interval is given by `risk_set`*$\hat{\rho}$.
+
+```{code-cell} ipython3
+mle_fit = weibull_min(c=2, scale=10_000)
+rho = mle_fit.cdf(600) - mle_fit.cdf(150) / (1 - mle_fit.cdf(150))
+print("Rho:", rho)
+print("N at Risk:", 1700)
+print("Expected Number Failing in between 150 and 600 hours:", 1700 * rho)
+print("Lower Bound 95% PI :", binom(1700, rho).ppf(0.05))
+print("Upper Bound 95% PI:", binom(1700, rho).ppf(0.95))
+```
+
+### Applying the Same Procedure on the Bayesian Posterior
+
+We'll use the posterior predictive distribution of the uniformative model. We show here how to derive the uncertainty in the estimates of the 95% prediction interval for the number of failures in a time interval.
+
+```{code-cell} ipython3
+def PI_failures(joint_draws, lp, up, n_at_risk):
```

Comment:
I'll update the code to use xarray-einstats, the inputs are xarray dataset and scalars IIUC, so there should be no need for looping in order to use scipy distributions.

Reply:
On this one... i think i kind of disagree. While it's not the most efficient way to do this operation, i wanted the parallel with the bootstrapping looping procedures to be clear. We're instantiating in a loop a reasonable model fit to the data in each iteration of the loop, and seeing the variation induced in downstream statistics... i think pedagogically this is simpler than hiding the similarity of the procedure in more efficient code....

Reply:
Oops I think I misread this comment yesterday, did you want to try and streamline the loop? My only point was that I'd like to keep the analogy with the bootstrap procedure clear....

But I think I could just add more text explaining that too.

Reply:
Ok, i've added some more text to highlight the relationship between the bootstrap and posterior predictive distributions if you want to work some xarray magic, that'd be cool! Thanks!

Reply:
Updated, and found a bug in xarray-einstats while doing it 😄 (I have fixed it already)

Reply:
Is it this one: 

<img width="763" alt="image" src="https://user-images.githubusercontent.com/10588167/212644548-dfb57b40-15d3-436e-b1b0-ae77cfb0c948.png">

I updated to my xarray einstats version, but still getting an error when i try to re-run your new function?

Reply:
Can recreate with just this call: 
<img width="690" alt="image" src="https://user-images.githubusercontent.com/10588167/212649410-32679f03-5fb6-4665-8c8c-95f8e788c5a6.png">

Reply:
Oh!!!! Sorry, i see the issue, the latest change is not on pip!!!! Sorry, slow this morning. 

<img width="362" alt="image" src="https://user-images.githubusercontent.com/10588167/212658370-fc1f356f-9b69-4aaa-a1b1-fbd0278a727c.png">

Ehm, that's cool. I'm happy with the changes and you can merge if you want. But for reproducibility should we be concerned that the user can't easily install the required packages?

Reply:
OK, cool! Let's do it. Thanks again for all your help on this one! I'm happy with it now if you are.